frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -169,284 +218,343 @@ export const ML_ALGORITHMS: Algorithm[] = [
-            zValidator: floatIntervalValidator([0,1], IntervalStrategy.CLOSED).optional()
+            zValidator: numberValidator('eta',
+                {   
+                    min: 0, 
+                    max: 1,
+                    isFloat: true 
+                })
         }, {
-            type: HyperparameterType.CONTINUOUS,
-            typeOptions: [HyperparameterType.CONTINUOUS, HyperparameterType.STATIC],
+            ...continuousParameterProperties,
             scalingType: 'Linear',
             key: 'gamma',
             value: ['0'],
             description: 'Minimum loss reduction required to make a further partition on a leaf node of the tree. The larger, the more conservative the algorithm is.',
-            zValidator: floatIntervalValidator([0,Number.POSITIVE_INFINITY], IntervalStrategy.RIGHT_OPEN).optional()
+            zValidator: numberValidator('gamma',
+                {   
+                    min: 0, 
+                    isFloat: true
+                }),
         }, {
-            type: HyperparameterType.INTEGER,
-            typeOptions: [HyperparameterType.INTEGER, HyperparameterType.STATIC],
+            ...integerParameterProperties,
             scalingType: 'Linear',
             key: 'max_depth',
             value: ['6'],
             description: 'Maximum depth of a tree. Increasing this value makes the model more complex and likely to be overfit. 0 indicates no limit. A limit is required when grow_policy=depth-wise.',
-            zValidator: integerIntervalValidator([0, Number.POSITIVE_INFINITY], IntervalStrategy.RIGHT_OPEN).optional()
+            zValidator: numberValidator('max_depth',
+                {   
+                    min: 0
+                }),
         }, {
-            type: HyperparameterType.CONTINUOUS,
-            typeOptions: [HyperparameterType.CONTINUOUS, HyperparameterType.STATIC],
+            ...continuousParameterProperties,
             scalingType: 'Linear',
             key: 'min_child_weight',
             value: ['1'],
             description: 'Minimum sum of instance weight (hessian) needed in a child. If the tree partition step results in a leaf node with the sum of instance weight less than min_child_weight, the building process gives up further partitioning. In linear regression models, this simply corresponds to a minimum number of instances needed in each node. The larger the algorithm, the more conservative it is.',
-            zValidator: floatIntervalValidator([0, Number.POSITIVE_INFINITY], IntervalStrategy.RIGHT_OPEN).optional()
+            zValidator: numberValidator('min_child_weight',
+                {   
+                    min: 0, 
+                    isFloat: true
+                })
```

Comment:
Another thing to check with the linter - you've got a combination of using a trailing comma and not using them.